x/oracle/keeper/upgrade.go

```diff
@@ -53,7 +53,22 @@ func (k Keeper) ApplyUpgrade(ctx sdk.Context, info *types.OracleUpgradeInfo) err
 	}
 	k.SetParams(ctx, params)
 
-	//TODO: update `Oracles` that already upgraded
+	iterator := k.GetOracleUpgradeQueueIterator(ctx, info.UniqueId)
+	defer iterator.Close()
+
+	for ; iterator.Valid(); iterator.Next() {
+		uniqueID, accAddr := types.SplitOracleUpgradeQueueKey(iterator.Key())
```

Comment:
It seems that we don't have to split the key.
Just use the value for address and use unique ID from upgrade info.